controllers/k8ssandra/vector.go

```diff
@@ -38,7 +38,7 @@ func (r *K8ssandraClusterReconciler) reconcileVector(
 
 		desiredVectorConfigMap := telemetry.BuildVectorAgentConfigMap(namespace, kc.SanitizedName(), dcConfig.Meta.Name, kc.Namespace, toml)
 		annotations.AddHashAnnotation(desiredVectorConfigMap)
-		k8ssandralabels.SetManagedBy(desiredVectorConfigMap, kcKey)
+		k8ssandralabels.SetWatchedByK8ssandraCluster(desiredVectorConfigMap, kcKey)
```

Comment:
We should be able to remove this as we already put the corresponding labels when creating the configmap in `BuildVectorAgentConfigMap()` [here](https://github.com/k8ssandra/k8ssandra-operator/blob/main/pkg/telemetry/vector.go#L163-L170).